packages/wallets/src/features/cfd/constants.tsx

```diff
@@ -96,7 +96,7 @@ export const getMarketTypeDetails = (
             title: getMarketTypeDetailsTitle(product, isEuRegion),
         },
         synthetic: {
-            availability: 'Non-EU',
+            availability: 'All',
```

Comment:
EU no longer limited to financial products since trading team side insisted that since DIEL financial should have synthetic instrument too